index.css

```diff
@@ -0,0 +1,167 @@
+.grid-container{
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-areas: 
+        'nav-bar'
+        'hero-section'
+        'product-banner'
+        'main-products';
+    height: auto;
+    width: 100vw;
+}
+
+.navigation-bar{
+    grid-area: nav-bar;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    min-height: 8rem;
```

Comment:
It will take the height of its content (child) which you already have defined as 8rem in logo class

Reply:
yes, implemented changes